pkgs/by-name/lu/lunacy/package.nix

```diff
@@ -0,0 +1,100 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, makeWrapper
+, autoPatchelfHook
+, zlib
+, libgcc
+, fontconfig
+, libX11
+, lttng-ust
+, icu
+, libICE
+, libSM
+, libXcursor
+, openssl }:
+
+stdenv.mkDerivation rec {
+  pname = "lunacy";
+  version = "9.3.3";
+
+  # see https://lunacy.docs.icons8.com/release-notes/
```

Comment:
Move this comment to a changelog attribute. See https://nixos.org/manual/nixpkgs/stable/#var-meta-changelog